src/Utils.js

```diff
@@ -329,6 +329,10 @@ class Utils {
    * @returns {object}
    */
   static clone(o) {
+    if (!o) {
+      // avoid calling clone on "undefined"
+      return o;
+    }
```

Comment:
curiosity : what bug does this fix ?

Reply:
ok answer in the description actually

Reply:
Bug without a ticket: if you get the status of a service that does not exist using pattyclient, pattyserver will throw because it will try to clone the state of a service that does not exists, i.e will try to clone "undefined".
Now we just return `undefined`, no cloning needed in that case.

Reply:
great thx.